scripts/overrides/tei_to_es.rb

```diff
@@ -148,28 +148,26 @@ def has_source
   end
 
   def spatial
-    get_elements(@xpaths["spatial"]).map do |ele|
+    get_elements(@xpaths["spatial"]).select{}.map do |ele|
```

Comment:
Does this `.select{}` here do anything when there are no arguments to it? Doesn't that just return the same array it was called on? Guessing this can be removed.

Reply:
I _think_ what I was trying to do ended up being covered in the code below. But see https://github.com/CDRH/family_letters/issues/181 for a somewhat related issue.

Reply:
Fix confirmed after re-posting data. Thanks!